APCAppCore/APCAppCore/Startup/APCAppDelegate.m

```diff
@@ -709,7 +711,8 @@ - (BOOL)handleUnsupportedAppVersionError:(NSError*)error networkManager:(id __un
 
 - (NSURL *)appStoreLinkURL
 {
-    return [[NSBundle mainBundle] appStoreLinkURL];
+    return [NSURL URLWithString:@""];
```

Comment:
And that is why pull requests are important!  I had project repo version conflicts and this statement wasn't compiling.  I changed it so I could keep developing.  Thanks for catching, it wasn't supposed to get committed.